client/src/components/FriendsRecommendations/index.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Button, Spinner } from 'reactstrap';
+import noAvatar from '../../assets/noAvatar.png';
+
+export interface IUser {
+    photoPath: string;
+    _id: string;
+    username: string;
+}
+
+export interface IFriendsRecommendation {
+    users: IUser[];
+    loading: boolean;
+}
+
+interface IProps {
+    loggedUsername: string;
+    friendsRecommendations: IFriendsRecommendation;
+    followUser: (body: {_id: string}) => void;
+}
+
+export const FriendsRecomendations = ({loggedUsername, friendsRecommendations, followUser}: IProps): JSX.Element => {
+    const topRecommendations = friendsRecommendations.users.slice(0, 4);
```

Comment:
why from 0 to 4?

Reply:
It's because on the feed we show only 4 first recommendations. There is also a button to show the rest recommendations on different page.

Reply:
4 in const

Reply:
Fixed.